packages/paste-core/components/pagination/stories/index.stories.tsx

```diff
@@ -225,9 +226,9 @@ export const ControlledNumberPaginationExample: React.FC = () => {
           disabled={currentPage === 1}
         />
         <PaginationNumbers pageLabel={`Page ${currentPage} of ${totalPages}`}>
-          {pages.map((page) => {
+          {pages.map((page, i) => {
```

Comment:
question: which rule is this for?

Reply:
react/jsx-key! You need to have a key when you're iterating over something: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md